dash/include/dash/GlobPtr.h

```diff
@@ -543,9 +488,57 @@ class GlobPtr
    */
   bool is_local() const {
     dart_team_unit_t luid;
-    dart_team_myid(_dart_gptr.teamid, &luid);
-    return _dart_gptr.unitid == luid.id;
+    DASH_ASSERT_RETURNS(
+      dart_team_myid(_rbegin_gptr.teamid, &luid),
+      DART_OK);
+    return _rbegin_gptr.unitid == luid.id;
+  }
+
+private:
+  void increment(index_type offs) {
```

Comment:
This sequence looks really ugly. I would suggest to modify the code  for better readabality with appropriate variables or even a small macros. Something like this:
~~~cpp
auto const & raddr = _rbegin_gptr.addr_or_offs.offset;
auto const lpos = _raddr / sizeof(value_type);
// Alternatively
#define _LPOS (addr) (addr / sizeof(element_type))
auto const lpos = _LPOS(r_addr);
~~~

I remember we decided to omit `dart_gptr_incaddr` since it does currently nothing but incrementing address and offset. This might be actually problematic IMHO. Will write something in #333